docs/source/reference/frameworks/torchscript.rst

```diff
@@ -0,0 +1,21 @@
+===========
+TorchScript
+===========
+
+.. admonition:: About this page
+
+   This is the API reference for TorchScript in BentoML.
+   You can find more information about TorchScript in the `official documentation <https://pytorch.org/docs/stable/jit.html>`_.
```

Comment:
Should we write a torchscript guide?

Reply:
I don't see why not. But it is probably not within the scope of this PR.